src/Core/Enums/InputMode.cs

```diff
@@ -0,0 +1,47 @@
+namespace Microsoft.FluentUI.AspNetCore.Components;
+
+/// <summary>
+/// The inputmode global attribute is an enumerated attribute that hints at the type of data that might be entered by the user while editing the element or its contents. This allows a browser to display an appropriate virtual keyboard. 
+/// </summary>
+public enum InputMode
+{
+    /// <summary>
+    /// No virtual keyboard. For when the page implements its own keyboard input control.
+    /// </summary>
+    None,
+
+    /// <summary>
+    /// Standard input keyboard for the user's current locale. (default value)
+    /// </summary>
+    Text,
+
+    /// <summary>
+    /// Fractional numeric input keyboard containing the digits and decimal separator for the user's locale (typically . or ,). Devices may or may not show a minus key (-).
+    /// </summary>
+    Decimal,
+
+    /// <summary>
+    /// Numeric input keyboard, but only requires the digits 0–9. Devices may or may not show a minus key. 
+    /// </summary>
+    Numeric,
+
+    /// <summary>
+    /// A telephone keypad input, including the digits 0–9, the asterisk (*), and the pound (#) key. Inputs that *require* a telephone number should typically use <see cref="TextFieldType.Tel"/> instead. 
+    /// </summary>
+    Tel,
```

Comment:
Could you use `Telephone` to avoid the abbreviation. It's not good practice to use an abbreviation that may be different for some people/cultures.

Reply:
You can add `using System.ComponentModel;` to your Enum and then add `[Description("...")]` attribute to each element to set the value that needs to be used in code (ie in this case 'tel' for 'Telephone'.
The `.ToAttributeValue()` Denis mentioned does that for you.